src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -214,11 +307,14 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
 
     // Main Header Apply button
     Button applyButton;
-    applyButton.Content(winrt::box_value(L"Apply"));
-    header.SetAlignRightWithPanel(applyButton, true);
-    header.SetLeftOf(cancelButton, applyButton);
+    applyButton.Content(winrt::box_value(L"OK"));
+    applyButton.Background(winrt::Windows::UI::Xaml::Media::SolidColorBrush{ WindowsColors::get_accent_color() });
```

Comment:
It looks like when you hover over the button it shows the default color. Can you check if this is an easy change? If not we can just track in a low priority issue.

Reply:
Unfortunately, it doesn't seem trivial to solve this. We would have to set a custom ControlTemplate for the Button. We should probably track this in a different issue.